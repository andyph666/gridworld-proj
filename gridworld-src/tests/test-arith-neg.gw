main{
	print(3 + -5);
	print(-5 - 4);
	print(4 * -6);
	print(12 % -5);
}