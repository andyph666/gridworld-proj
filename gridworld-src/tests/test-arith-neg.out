-2
-9
-24
-3